CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+# Changed
+
+- `view` default Action does not nest an `<a>` tag anymore.
```

Comment:
this is a breaking change right? i mean it will still compile but probably projects have to update or else their links are not working anymore? if yes, then could you may add the fancy explosion icon in front.. like we did here:
https://github.com/neolution-ch/react-hook-form-components/releases/tag/1.0.1
many thanks :)

Reply:
@neotob Done :D

Reply:
awesome, thank you! :)